package.json

```diff
@@ -30,10 +30,13 @@
     "@tailwindcss/postcss7-compat": "^2.2.17",
     "@types/greensock": "^1.15.32",
     "core-js": "^3.25.0",
+    "expr-eval": "^2.0.2",
     "gsap": "^3.11.1",
     "harp": "^0.41.0",
+    "i": "^0.3.7",
```

Comment:
yo, what's this library for and where are you using it?

Reply:
wow this is funky. I dont think I;ve installed anyrthing of the sort